packages/terminology/src/sqlite.ts

```diff
@@ -0,0 +1,72 @@
+import { join } from "path";
+import { writeFileSync, existsSync } from "fs";
+
+import { S3Utils } from "./s3";
+
+import { TermServerClient, createTermServerClient } from "./sqlClient";
+import * as migrations from "./migrations/index";
+import { seedCodeSystems } from "./seed/seedCodeSystem";
+import { Config } from "./config";
+
+const region = Config.getAWSRegion();
+
+let dbClient: TermServerClient | undefined;
```

Comment:
We should use the more abstract representation, so we can really leverage the fact we have an interface in case we want to switch implementations.